foundry-bin/update.sh

```diff
@@ -43,7 +44,7 @@ get_hash() {
 
 cat > releases.nix << EOF
 {
-  version = "0.0.0";
+  version = "gh-node-id-${node_id}";
```

Comment:
Let's prefix this with something like `0.0-` just so it's still sorted correctly after we someday start bumping versions.

Reply:
Maybe just `0.0-$hash`

Reply:
@shazow if we wanted to do a pseudo semver it would be like `0.0.0-$hash` something along the lines of `-rc` or `-beta`

Reply:
Sounds good to me.